packages/framework/src/Migrations/Version20240318162418.php

```diff
@@ -14,6 +14,14 @@ class Version20240318162418 extends AbstractMigration
      */
     public function up(Schema $schema): void
     {
+        $postgresqlVersion = $this->sql('SELECT version();')->fetchOne();
+        $postgresqlVersion = substr($postgresqlVersion, 11, 2);
+        $existsNormalizeFunction = $this->sql('SELECT 1 FROM pg_proc WHERE proname = \'normalize\'')->fetchOne();
+
+        if ($postgresqlVersion > 12 || $existsNormalizeFunction !== 1) {
+            return;
+        }
```

Comment:
the source of the problem is probably in the `Version20180603135340.php` migration, where the normalize function is created. 

I have tested it with postgres 13.0 and I was not able to create database schema due to normalize function in that migration. Can you please check it?

Reply:
ups, I forgot to commit changes in `Version20180603135340.php` migration, sorry. Fixed in https://github.com/shopsys/shopsys/pull/3072/commits/491a0f973ab716594bba2c8188da414155da7a82